types/order.ts

```diff
@@ -11,6 +11,12 @@ export interface BillingAddress {
     country: string;
 }
 
+export interface OrderProduct {
+    name: string;
```

Comment:
We should add `[key: string]: unknown;` to show that there are additional fields to this interface.